examples/shader/cpu_texture.rs

```diff
@@ -0,0 +1,168 @@
+use bevy::{
+    prelude::*,
+    reflect::TypeUuid,
+    render::{
+        pipeline::{PipelineDescriptor, RenderPipeline},
+        render_graph::{base, AssetRenderResourcesNode, RenderGraph},
+        renderer::RenderResources,
+        shader::{ShaderStage, ShaderStages},
+        texture::{ TextureDimension, TextureFormat, Extent3d },
+    },
+    utils::Duration,
+};
+use std::num::Wrapping;
+
+/// This example illustrates how to mutate a texture on the CPU.
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_asset::<MyTexture>()
+        .add_startup_system(setup.system())
+        .add_startup_system(setup_timer.system())
+        .add_system(timer_tick.system())
+        .run();
+}
+
+/// The component that represents the mutable texture.
+/// It is a resource at the same time.
+#[derive(RenderResources, Default, TypeUuid)]
+#[uuid = "93fb26fc-6c05-489b-9029-601edf703b6b"]
```

Comment:
Could you add a comment on where this `uuid` comes from/why it is necessary?

Reply:
Done. I've actually taken the UUID from the array_texture example, so I generated a new one here.